OpenRA.Mods.RA/Guard.cs

```diff
@@ -18,10 +18,22 @@
 namespace OpenRA.Mods.RA.Traits
 {
 	[Desc("The player can give this unit the order to follow and protect friendly units with the Guardable trait.")]
-	class GuardInfo : TraitInfo<Guard> { }
+	class GuardInfo : ITraitInfo
+	{
+		public readonly string VoicePhrase = "Move";
```

Comment:
This field should be simply called `Voice`. See C4Demolition.cs, Selectable.cs, etc